modules/ui/src/main/scala/lucuma/ui/forms/EnumViewOptionalSelect.scala

```diff
@@ -122,8 +122,7 @@ final case class EnumViewOptionalSelect[F[_], A](
 
   override def setter(ddp: FormDropdown.FormDropdownProps): Callback =
     ddp.value.toOption
-      .flatMap(v => enum.fromTag(v.asInstanceOf[String]))
-      .map(v => value.set(Some(v)).runAsyncCB)
+      .map(v => value.set(enum.fromTag(v.asInstanceOf[String])).runAsyncCB)
```

Comment:
Can it happen that `ddp.value.toOption` is `None` for unsetting?

Maybe we can foresee this case and always call `value.set(ddp.value.toOption.flatMap(enum.fromTag)`

Reply:
When you unset you get an empty string